File: app/src/main/java/com/example/zukkey/arcoresampleforprimer/java/CloudAnchorFragment.java
package com.example.zukkey.arcoresampleforprimer.java;

import com.google.ar.core.Config;
import com.google.ar.core.Session;
import com.google.ar.sceneform.ux.ArFragment;

public class CloudAnchorFragment extends ArFragment {

  @Override
  protected Config getSessionConfiguration(Session session) {
    Config config = super.getSessionConfiguration(session);
    config.setCloudAnchorMode(Config.CloudAnchorMode.ENABLED);
    return config;
  }
}

File: app/src/main/java/com/example/zukkey/arcoresampleforprimer/kotlin/CloudAnchorFragment.kt
package com.example.zukkey.arcoresampleforprimer.kotlin

import com.google.ar.core.Config
import com.google.ar.core.Session
import com.google.ar.sceneform.ux.ArFragment


class CloudAnchorFragment : ArFragment() {

  override fun getSessionConfiguration(session: Session): Config {
    return super.getSessionConfiguration(session).apply {
      cloudAnchorMode = Config.CloudAnchorMode.ENABLED
    }
  }
}
